src/app.js

```diff
@@ -104,7 +109,10 @@ adminAppRouter.get(`${openapi.basePath}`, async (req, res) => {
 initialize({
   app: appRouter,
   apiDoc: openapi,
-  paths: `${appRoot}/api${openapi.basePath}/paths`,
+  paths: [
+    { path: '/pets', module: pets },
+    { path: '/pet', module: pet },
+  ],
```

Comment:
As we discussed, this method will not work with bundling since it creates runtime dependence on the paths directory existing. The current method signals to webpack to bundle those modules, and associates those modules with the correct paths in express.